client/signup/steps/import-url/index.jsx

```diff
@@ -120,6 +120,11 @@ class ImportURLStepComponent extends Component {
 		this.props.fetchIsSiteImportable( this.props.urlInputValue );
 	};
 
+	setInputValueFromQueryArg = () => {
+		const urlFromQueryArg = get( this.props, 'queryObject.url' );
+		urlFromQueryArg && this.props.setNuxUrlInputValue( urlFromQueryArg );
```

Comment:
Just my personal opinion: I'm not a fan of this syntax outside of jsx templates because I find it less readable than a plain `if () {}` block.